### PR TITLE
Add class methods counter in xml output

### PR DESCRIPTION
**Pull request by lucaskanashiro**
Signed-off-by: Lucas Kanashiro <kanashiro@debian.org>
 Signed-off-by: Giuliano Belinassi <giuliano.belinassi@usp.br>

### CODE REVIEW

File: lib/kuniri/parser/xml_output_format.rb [hunk 1]
```diff
@@ -23,26 +23,39 @@ def class_generate(pClass)
...
           wrapper.inheritance_generate(singleClass.inheritances)
-          unless singleClass.attributes.empty?
-            wrapper.attribute_generate(singleClass.attributes)
-          end
+          wrapper.handle_attributes(singleClass.attributes)
+
           singleClass.constructors.each do |singleConstructor|
             wrapper.function_behaviour_generate('constructorData',
                                                 singleConstructor)
+            methods_counter += 1
```
Comment (rodrigosiqueira) [inline lib/kuniri/parser/xml_output_format.rb]: The constructor and methods are two different things. It does not make sense to count a constructor like a method. E.g.: One class can have multiple constructors and multiple methods. Remove the counter from this part, maybe you can send another PR with the constructor part.

**Review by rodrigosiqueira — COMMENTED** [review]
flagEnglisharrowgif
Finally, I did not find any test to verify the total of method in different situations. For example, It is a good idea to add one test that verify three our four classes with different number of methods and check if the final value is correct or not.

**Comment by giulianobelinassi** [comment]
Fixed it. Also added a test that checks it.
